packages/page_object/lib/src/page_object.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+/// This class exists so page objects can seamlessly be passed to widget testing APIs as Finders. This helps
+/// to remove code noise from tests. See the tests in this package for an example.
+class PageObject extends Finder {
```

Comment:
So, with this... the generic GWT Harness... and presumably a set of GWT assertions to wrap widget tester...

how micro do we want to on these packages? I'm of two minds:

1) bundle them together
2) package them separately, but then offer a composite package that bundles them together?

I'm just not convinced at how much utility they provide individually? thoughts?

Reply:
While we use all of these tools together I view them as completely separate functionality that other teams might only want pieces of. The friction to maintain them and use them separately is very low. If we wanted to create a flutter glove box package that includes all of these I would be open to that.

Regarding the collection GWT assertions, I could see other teams wanting very different names or similar names but different signatures. I'd prefer not to be overly opinionated about that API.